understanding/22/focus-appearance-minimum.html

```diff
@@ -49,7 +49,7 @@ <h2>Focus Appearance (Minimum) Success Criterion text</h2>
             </dd>
             <dt class="new"><dfn>minimum bounding box</dfn></dt>
             <dd class="new">
-                the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie. Where a component consists of disconnected parts, such as a link that wraps onto multiple lines, each part is considered to have its own bounding box.
+                the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie. Where a component consists of disconnected parts, such as a link that wraps onto multiple lines, each part is considered to have its own bounding box. In such a case, the base for further measurement is the concatenation of all boxes for disconnect parts. In the example of a link that wraps onto multiple lines the resulting box would enclose the same link if it would not wrap.
```

Comment:
`In the example of a link that wraps onto multiple lines the resulting box would enclose the same link if it would not wrap.`

perhaps adding a comma, and changing `would` to `did`?

`In the example of a link that wraps onto multiple lines, the resulting box would enclose the same link if it did not wrap.`